src/openeo_aggregator/backend.py

```diff
@@ -675,74 +675,120 @@ def _get_connection_and_backend_service_id(
+
+            It is a dict in order to keep it simple to cache this information.
+
+            1. "supported_backend_ids" maps to a list containing the backend IDs for
+                backends that support secondary services.
+                An aggregators may have backends that do not support secondary services.
+
+            2. "service_types" maps to a dict with two items:
+                - backend_id:
+                    which backend provides that service type
+                - service_type:
+                    the info that this backend reports on its GET /service_types endpoint,
+                    about that service type.
+
+            For example:
+
+            {
+                "supported_backend_ids": ["b1", "b2"]
+                "service_types": {
+                    "WMTS": {
+                        "backend_id": "b1",
+                        "service_type":  # contains what backend b1 returned for service type WMTS.
+                        {
+                            "configuration": "..."
+                        }
+                    },
+                    "WMS": {
+                        "backend_id": "b2",
+                        "service_type": {
+                            "configuration": "..."
+                        }
+                    }
                 }
             }
-         'WMS':
-            {'backend_id': 'b2',
-             'service_type':
-                {'configuration':
-                    ...
-                }
-            }
-        }
 
-        Info for selecting the right backend to create a new service
-        ------------------------------------------------------------
+        Assumptions: Selecting the right backend to create a new service
+        ----------------------------------------------------------------
 
         We are assuming that there is only one upstream backend per service type.
         That means we don't really need to merge duplicate service types, i.e. there is no need to resolve
         some kind of conflicts between duplicate service types.
 
         So far we don't store info about the backends' capabilities, but in the future
-        we may need to store that to select the right backend when we create a service.
+        we may need to take into account configuration to select the right backend when
+        we create a service. For now we just ignore it.
 
         See: issues #83 and #84:
             https://github.com/Open-EO/openeo-aggregator/issues/83
             https://github.com/Open-EO/openeo-aggregator/issues/84
         """
 
-        # TODO: Issue #85 data about backend capabilities could be added to the service_types data structure as well.
+        # Along with the service types we query which backends support secondary services
+        # so we can cache that information.
+        # Some backends don not have the GET /service_types endpoint.
+        supported_backend_ids = self._find_ids_supported_backends()
```

Comment:
as noted above: I would just inline `_find_ids_supported_backends` here, 
or even just build the list inside the for loop a couple of lines below

Reply:
Done.